esrally/rally.py

```diff
@@ -587,6 +587,15 @@ def dispatch_sub_command(cfg, sub_command):
         return False
 
 
+def to_dict(arg):
+    if io.has_extension(arg, ".json"):
+        import json
```

Comment:
Minor comment: you probably have your reasons, but why not have this at the top? AFAIK [pep8](https://www.python.org/dev/peps/pep-0008/#imports) is rather strict about this. I read some literature preferring `from json import load` for inside-function uses such as this one, if it must be done.

Reply:
I wanted to limit the scope of the import so we don't accidentally use this functionality somewhere else. I'm fine with putting it at the top. Mid-term it might make sense to split `rally.py` though.